### PR TITLE
Update UltraPunishments_Italian.lang

### PR DESCRIPTION
**Pull request by FranFrau**
Translated all messages

### CODE REVIEW

File: Translations/UltraPunishments_Italian.lang [hunk 1]
```diff
@@ -1,47 +1,47 @@
...
-#GUI_PLAYERVIEW_EXCLUDE_ACTION: Click to %1 punishments for this player
-#GUI_OVERVIEW_TEMPLATES_ACTION: Click to show templates
-#GUI_PLAYERVIEW_PUNISH_UNPUNISHABLE: Punishments for this player have been disabled.
-#TYPE_IN_MESSAGE: Type in a message
-#SET_A_DURATION: Click to set duration
-#PLAYERS: Players
-#REASON: Reason
+#GUI_PLAYERVIEW_EXCLUDE_ACTION: Fai click su %1 per punire questo giocatore
+#GUI_OVERVIEW_TEMPLATES_ACTION: Clicca per mostrare i template
+#GUI_PLAYERVIEW_PUNISH_UNPUNISHABLE: Le punizioni per questo giocatore sono disabilitate.
```
Comment (rlnt) [inline Translations/UltraPunishments_Italian.lang]: Remove the dot at the end. It's misplaced in the original translation and not needed here

**Review by rlnt — CHANGES_REQUESTED** [review]
Change this one thing and it's fine, thanks for submitting

**Comment by timokueck** [comment]
I got rid of that dot. Thanks for contributing Pompiere1!

**Review by rlnt — COMMENTED** [review]
Thanks

**Pull request merged**